nmap.cc

```diff
@@ -709,6 +709,7 @@ void parse_options(int argc, char **argv) {
       if (strcmp(long_options[option_index].name, "script") == 0) {
         o.script = 1;
         o.chooseScripts(optarg);
+        o.script_expression = strdup(optarg);
```

Comment:
Instead of creating another option containing a copy of the `--script`, let's just remove the `chooseScripts` function. That function is just splitting optarg by commas which is not really necessary (Lua can do that just as easily). And now with this new change, splitting by commas in C++ is counterproductive as NSE needs to specially handle the parsing of `--script`.

This will require changes in `nse_main.cc` to pass the `--script` argument to `nse_main.lua`.